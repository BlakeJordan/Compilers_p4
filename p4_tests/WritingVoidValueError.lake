void f() {
    write "hehehehaha";
}

int g() {
    write f();
}