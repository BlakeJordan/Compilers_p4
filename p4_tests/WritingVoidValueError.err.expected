6,11: Attempt to write void
Type checking failed
